testing/tests/DevExpress.ui.widgets.form/form.materialTheme.tests.js

```diff
@@ -0,0 +1,89 @@
+import $ from 'jquery';
+import 'ui/text_area';
+import 'ui/autocomplete';
+import 'ui/calendar';
+import 'ui/date_box';
+import 'ui/drop_down_box';
+import 'ui/html_editor';
+import 'ui/lookup';
+import 'ui/radio_group';
+import 'ui/tag_box';
+
+import 'common.css!';
+import 'material_blue_light.css!';
+
+QUnit.testStart(function() {
+    const markup = '<div id="form"></div>';
+
+    $('#qunit-fixture').html(markup);
+});
+
+QUnit.module('Form -> material theme');
+
+QUnit.test('Form have correct padding (T849353)', function(assert) {
+    const $form = $('#form').dxForm({
```

Comment:
This test with a big layout of Form. May be, to separate this on the different tests on a different scenarios.

Reply:
Fixed